Formula/trojan.rb

```diff
@@ -0,0 +1,49 @@
+class Trojan < Formula
+  desc "Unidentifiable mechanism that helps you bypass GFW"
+  homepage "https://trojan-gfw.github.io/trojan/"
+  url "https://github.com/trojan-gfw/trojan/archive/v#{version}.tar.gz"
+  version "1.10.1"
+  sha256 "7524fbff42013528dd19b2202cab0ea24085cbdc153d394d086e6bf8294b889e"
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :test
+  depends_on "python" => :test
+  depends_on "boost"
+  depends_on "openssl@1.1"
+
+  def install
+    system "sed", "-i", "", "s/server\\.json/client.json/", "CMakeLists.txt"
```

Comment:
- Why are the replacements necessary? They should be submitted upstream for fixing.
- Use `inreplace` rather than calling `sed`

Reply:
> * Why are the replacements necessary? They should be submitted upstream for fixing.

Trojan can run as both server and client, and upstream provides a server side config by default. Since most macOS users run trojan in client mode, the replacement changes the default config file to a client side one.

> * Use `inreplace` rather than calling `sed`

Fixed.